src/renderer.js

```diff
@@ -207,10 +208,13 @@ class Markdown {
       });
     }
 
-    let children = node.nodes.map(node => this.serializeNode(node, document));
-    children = children.flatten().length === 0
-      ? ""
-      : children.flatten().join("");
```

Comment:
Is there a particular reason you removed the flatten usage?

Reply:
Yeah.  The flatten logic was essentially joining everything with `""`.  To get the multi-line blockquote use case to work, we needed to use `""` to join the inner elements but use `"\n"` to join nodes.